backend/oas/provider/openapi.yaml

```diff
@@ -321,6 +321,53 @@ paths:
                 $ref: '#/components/schemas/error.NotFoundError'
               example:
                 message: Job not found
+  /jobs/{job_id}/transpiler_info:
+    put:
```

Comment:
This is a PUT endpoint, not a PATCH.
This is because, in general, **PUT is used for endpoints that define the GET response for the corresponding resource**.
When `transpiler_info` is considered a subresource of the selected job,
a request to this endpoint defines the entire structure of the resource.